syntax = "proto3";

package NGenerativeBoltalka.Proto;

message TBertFactorRequest {
    repeated string Context = 1;
    repeated string Candidates = 2;
}
